src/main/java/com/akamai/edgeauth/EdgeAuth.java

```diff
@@ -350,7 +366,7 @@ public void setTokenType(String tokenType) {
      * @throws EdgeAuthException EdgeAuthException
      */
     public void setTokenName(String tokenName) throws EdgeAuthException {
-        if (tokenName == null || tokenName == "") {
+        if (tokenName == null || tokenName.isEmpty()) {
```

Comment:
`isEmpty()` returns true if there are any characters present, so for the string "  " (with space/s) it will return false. Do we want to cover that case at all?

Reply:
If yes, we can use `stringVariable.trim().isEmpty()`

Reply:
Good idea, applied.